suites/load-testing/grafana/grafana-provisioning/datasources/datasource.yml

```diff
@@ -0,0 +1,11 @@
+apiVersion: 1
+datasources:
+  - name: InfluxDB
+    type: influxdb
+    access: proxy
+    database: db0
+    user: admin
+    password: admin
+    url: http://influxdb:8086
+    isDefault: true
```

Comment:
Let's not have a token.txt file in the repo.  We don't want to set up people to accidentally checkin tokens.  Change the scripts to load access tokens via environment variables.